contents/docs/product-analytics/toolbar.mdx

```diff
@@ -8,38 +8,147 @@ availability:
+
+- **Selector:** Path to the element on the page. If you're not happy with the selector PostHog is using automatically, you could, for example, select elements based on their `id` or `data-attr`. You can type a selector into the "Selector" field in the action creation modal to do this.
+
+![The create action modal showing the selector text box being used to edit the selector](../../images/tutorials/toolbar/type-it.gif)
+
+Or in the same modal, click "Edit the selector" to use our element picker to build the selector you want.
+
+![The selector modal being used to discover the more meaningful selector](../../images/tutorials/toolbar/toolbar-selector-edit.gif)
+
+## Viewing your existing actions
+
+You can view your existing actions on your page by clicking the "Actions" button on the Toolbar. This will highlight any elements that are associated with an action. This is a useful way to check that elements are associated with the correct actions.
+
+By clicking on a specific element, you can update or delete the action.
+
+Note that the number in the green box on the top left of the element is the action's index in your list of actions (and does not represent how many times that action was triggered). The number in the yellow box next to the "Actions" button in the toolbar represents the total number of actions you have created.
+
+![How to setup the PostHog toolbar](../../images/products/product-analytics/toolbar-view-actions.mp4)
+
+## Overriding feature flags with the toolbar
+
+The toolbar also enables you to test your feature flags. You can enable, disable, or override your feature flags, and then view how your website or app changes with the new feature flags values.
+
+To do this, click on the "Feature Flags" button in the toolbar. Then, search for any feature flag of choice and click on the toggles to change it's value. 
+
+Note that overriding feature flags will only affect *your* browser. You may also need to refresh the page to see how your change affect your website.
+
+![How to override feature flags using the tool](../../images/products/product-analytics/toolbar-feature-flags.mp4) 
+
+## Troubleshooting and FAQ
 
-## Toolbar not loading or displaying?
+### Toolbar not loading or displaying
 
 When launched, the toolbar injects an API token in the format `#__posthog=JSON` in your site's URL, like this: `http://mysite.com/#__posthog=JSON`.
 
 The toolbar won't load if you have a frontend that overrides the injected API token (the information after the #) before the `posthog-js` library (or snippet) loads and has a chance to read the API token.
 
-To solve this, get the `__posthog` JSON from the URL before it is overridden and call `posthog.loadToolbar(JSON)`:
+To solve this, retrieve the `__posthog` JSON from the URL before it is overridden and call `posthog.loadToolbar(JSON)`:
```

Comment:
hmm good point. I actually tried to do this now and was not able to. @pauldambra can share more details on how to do this exactly?

I tried in the browser console, but ran into "posthog not defined" issue

Reply:
I think @mariusandra implemented this?

Reply:
You'd call this in your code, wherever you have access to this variable in your project's frontend router.

I think in our case, `posthog` is not available in the console because we use the npm package.